src/compiler/checker.ts

```diff
@@ -23075,6 +23065,11 @@ export function createTypeChecker(host: TypeCheckerHost): TypeChecker {
         return isArrayLikeType(type) || isTupleLikeType(type);
     }
 
+    function isAssignableToAvailableAnyIterable(type: Type): boolean {

```

Comment:
Maybe I'm brainfarting, but what does "Available" mean in this context?

Reply:
When we are in es5 the "available any iterable" becomes `anyReadonlyArray`. I know that the name isn't the best but I wanted to indicate that it might not exactly be testing against `anyIterable` (cause that might not be available). I'm open to suggestions for a better name here